hugegraph-core/src/main/java/com/baidu/hugegraph/job/computer/AbstractComputer.java

```diff
@@ -162,16 +170,22 @@ private void initializeConfig(ComputerJob job) throws Exception {
     }
 
     private Map<String, Object> readSubConfig(String sub) {
-        List<ConfigurationNode> nodes = this.config.getRootNode()
-                                                   .getChildren(sub);
-        E.checkArgument(nodes.size() == 1,
-                        "Must contain one '%s' node in config file '%s'",
-                        sub, this.config.getFileName());
-
-        List<ConfigurationNode> subConfigs = nodes.get(0).getChildren();
-        Map<String, Object> results = new HashMap<>(subConfigs.size());
-        for (ConfigurationNode node : subConfigs) {
-            results.put(node.getName(), node.getValue());
+        List<HierarchicalConfiguration<ImmutableNode>> nodes =
+        this.config.childConfigurationsAt(sub);
+
+        E.checkArgument(nodes.size() >= 1,
+                        "Node '%s' must contain", sub);
+
+        ImmutableNode root = null;
+        NodeModel<ImmutableNode> nodeModel = null;
+        NodeHandler<ImmutableNode> nodeHandler = null;
+        Map<String, Object> results = new HashMap<>(nodes.size());
+        for (HierarchicalConfiguration<ImmutableNode> node : nodes) {
+            E.checkArgument((nodeModel = node.getNodeModel()) != null &&
+                           (nodeHandler = nodeModel.getNodeHandler()) != null &&
```

Comment:
can move `nodeModel = node.getNodeModel()` to independent line for better readability